reconcile/utils/openshift_resource.py

```diff
@@ -94,6 +94,13 @@ def obj_intersect_equal(self, obj1, obj2):
                     ]
                     if diff or not self.obj_intersect_equal(obj1_v, obj2_v):
                         return False
+                elif obj1_k == "env":
+                    for v in obj2_v:
+                        if "name" in v and len(v) == 1:
```

Comment:
if the length is 1 it means the `name` is the only key, but this is more readable IMO